src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -37,13 +37,15 @@ export interface IHeaderPlaceholders {
 
 export interface ItemListLayoutProps<T extends ItemObject = ItemObject> {
   tableId?: string;
+  clusterId?: string;
```

Comment:
This component shouldn't care about what cluster it is in, and wouldn't need to if you use `createStorage`